test/utils.test.js

```diff
@@ -89,6 +89,7 @@ describe('utils', function() {
     describe('isErrorEvent', function() {
       it('should work as advertised', function() {
         assert.isFalse(isErrorEvent(new Error()));
+        assert.isFalse(isError(new ErrorEvent('')));
```

Comment:
This is `isError` test, not `isErrorEvent` and should be moved to line ~122 w. additional `supportsErrorEvent()` clause, eg. `if (supportsErrorEvent()) assert.isFalse(isError(new ErrorEvent('')));`

Reply:
Done!